src/garage/torch/modules/gaussian_mlp_module.py

```diff
@@ -121,12 +121,13 @@ def __init__(self,
         if self._std_parameterization not in ('exp', 'softplus'):
             raise NotImplementedError
 
-        init_std_param = torch.Tensor([init_std]).log()
+        self._init_std = torch.Tensor([init_std]).log()
+        log_std = torch.Tensor([init_std] * output_dim).log()
```

Comment:
don't think a log() is necessary here, since it has already been taken in line 124?